config/schedule.rb

```diff
@@ -0,0 +1,12 @@
+every 1.day do
```

Comment:
Given the idempotence of these rake tasks I'm worried about the likelihood of spamming the same message multiple times when "things happen".  We might end up needing some sort of way to track which ones have been successfully sent.